3-UsingAPIs/Week1/homework/ex3-rollDice.js

```diff
@@ -42,3 +42,6 @@ rollDice((error, value) => {
 
 // ! Do not change or remove the code below
 module.exports = rollDice;
+
+// In promise version => if there is an error we don't get `Success!`
+// In callback version => if there is an error we always get `Success!`
```

Comment:
You are describing what happens, not why it differs: the outcome of a promise, once settled (resolved or rejected) can no longer be changed. Subsequent calls to resolve() or reject() have no effect.